src/api-docs/source/includes/_wallet_rpc.md

```diff
@@ -0,0 +1,2858 @@
+}
+
+const walletClient = new WalletClient(walletOptions);
+
+(async () => {
+  const result = await walletClient.execute('resendwallettransactions');
+  console.log(result);
+})();
+```
+
+> The above command returns JSON "result" like this:
+
+```json
+[
+  "147c2527e2bb7ddfa855cc4b933ab288e05aa7816c487db69db344971f1b0951",
+  "c3c92d6686442755d70d2ea44401437d9fab51bc7a504b041d6d6b950ba45e85",
+  "77f09f2f307aaa62c8d36a9b8efeac368381c84ebd195e8aabc8ba3023ade390",
+  "2c0fa5740c494e8c86637c1fad645511d0379d3b6f18f84c1e8f7b6a040a399c",
+  "ef38a6b68afe74f637c1e1bc605f7dc810ef50c6f475a0a978bac9546cac25d8",
+  "1146d21bb5c46f1de745d9def68dafe97bbf917fe0f32cef31937731865f10e9"
+]
+```
+
+Re-broadcasts all unconfirmed transactions to the network.
+
+### Params
+N. | Name | Default |  Description
+--------- | --------- | --------- | -----------
+None. |
+
+
+
+## abandontransaction
+
+```javascript
+let tx;
+```
+
+```shell--vars
+tx='a0a65cd0508450e8acae76f35ae622e7b1e7980e95f50026b98b2c6e025dae6c'
+```
+
+```shell--curl
+curl $walletrpcurl \
+  -X POST \
+  --data '{
+    "method": "abandontransaction",
+    "params": [ "'$tx'" ]
+  }'
+```
+
+```shell--cli
+hsw-rpc abandontransaction $tx
+```
+
+```javascript
+const {WalletClient} = require('hs-client');
+const {Network} = require('hsd');
+const network = Network.get('regtest');
+
+const walletOptions = {
+  network: network.type,
+  port: network.walletPort,
+  apiKey: 'api-key'
+}
+
+const walletClient = new WalletClient(walletOptions);
+
+(async () => {
+  const result = await walletClient.execute('abandontransaction', [tx]);
+  console.log(result);
+})();
+```
+
+> The above command returns JSON "result" like this:
+
+```json
+null
+```
+
+Remove transaction from the database. This allows "stuck" coins to be respent.
+
+### Params
+N. | Name | Default |  Description
+--------- | --------- | --------- | -----------
+1 | txid | Required | Transaction ID to remove
+
+
+## backupwallet
+
+```javascript
+let path;
+```
+
+```shell--vars
+path='/home/user/WalletBackup'
+```
+
+```shell--curl
+curl $walletrpcurl \
+  -X POST \
+  --data '{
+    "method": "backupwallet",
+    "params": [ "'$path'" ]
+  }'
+```
+
+```shell--cli
+hsw-rpc backupwallet $path
+```
+
+```javascript
+const {WalletClient} = require('hs-client');
+const {Network} = require('hsd');
+const network = Network.get('regtest');
+
+const walletOptions = {
+  network: network.type,
+  port: network.walletPort,
+  apiKey: 'api-key'
+}
+
+const walletClient = new WalletClient(walletOptions);
+
+(async () => {
+  const result = await walletClient.execute('backupwallet', [path]);
+  console.log(result);
+})();
+```
+
+> The above command returns JSON "result" like this:
+
+```json
+null
+```
+
+Back up wallet database and files to directory created at specified path.
+
+### Params
+N. | Name | Default |  Description
+--------- | --------- | --------- | -----------
+1 | path | Required | Path and name of destination directory for backup
```

Comment:
suggestion: "Path and name" here may be confusing. Perhaps "Path to backup's destination directory."